app/com/gu/contentapi/sanity/ElasticSearchSnapshotTest.scala

```diff
@@ -0,0 +1,22 @@
+package com.gu.contentapi.sanity
+
+import com.gu.contentapi.sanity.tags.ProdOnly
+import com.amazonaws.services.s3.AmazonS3Client
+import java.time.LocalDate
+
+@ProdOnly
+class ElasticSearchSnapshotTest(context: Context) extends SanityTestBase(context) {
+
+    "Elasticsearch backup via snapshot" should "have been updated yesterday on S3" in {
+
+      val yesterdayDate = LocalDate.now().minusDays(1L)
+      val objectKey = s"PROD-ZEBRA/meta-${yesterdayDate}-00-00.dat"
```

Comment:
`PROD-ZEBRA` will change to `PROD-AARDVARK` when we switch stacks (which we do when we upgrade Elasticsearch) meaning this test will break.